app/containers/ActionEdit/index.js

```diff
@@ -68,29 +74,77 @@ export class ActionEdit extends React.PureComponent { // eslint-disable-line rea
           </div>
         }
         {action &&
-          <EntityForm
-            model="actionEdit.form.action"
-            handleSubmit={this.props.handleSubmit}
-            fields={
-              [
-                {
-                  id: 'title',
-                  controlType: 'input',
+          <Page
+            title={this.context.intl.formatMessage(messages.pageTitle)}
+            actions={[
+              {
+                type: 'simple',
+                title: 'Cancel',
+                onClick: this.props.handleCancel,
+              },
+              {
+                type: 'primary',
+                title: 'Save',
+                onClick: () => this.props.handleSubmit(this.props.form.action),
+              },
+            ]}
+          >
+            <EntityForm
+              model="actionEdit.form.action"
+              handleSubmit={this.props.handleSubmit}
+              handleCancel={this.props.handleCancel}
+              fields={{
+                header: {
+                  main: [
+                    {
+                      id: 'title',
+                      controlType: 'input',
+                      model: '.title',
+                      validators: {
+                        required,
+                      },
+                      errorMessages: {
+                        required: this.context.intl.formatMessage(messages.fieldRequired),
+                      },
+                    },
+                  ],
+                  aside: [
+                    {
+                      id: 'no',
+                      controlType: 'info',
+                      displayValue: reference,
+                    },
+                    {
+                      id: 'status',
+                      controlType: 'select',
+                      model: '.draft',
+                      value: action && action.draft,
+                      options: PUBLISH_STATUSES,
+                    },
+                    {
+                      id: 'updated',
+                      controlType: 'info',
+                      displayValue: action && action.get('attributes').get('updated_at'),
+                    },
+                    {
+                      id: 'updated_by',
```

Comment:
added new (non-)`controlType` info here for fields that are display only and not editable